WIP-CHANGELOG.md

```diff
@@ -38,6 +39,7 @@ The following options has been removed or replaced
 - Support for GCS
 - Backup retention for InfluxDB.
 - Add Okta as option for OIDC provider
+- Added record `cluster.name` in all logs to elasticsearch that matches the cluster setting `global.environmentName`
```

Comment:
```suggestion
- Added record `cluster.name` in all logs to elasticsearch that matches the cluster setting `global.clusterName`
```

Reply:
Since it's separate commits I thought this would go first and add the fluentd stuff and the next commit changes the name to clusterName

Reply:
While it makes sense when reviewing the commits, I think this makes the changelog confusing to read. I would change this line in the wip changelog also as part of the second commit. So you actually rename it everywhere, also in the changelog. :slightly_smiling_face:

Reply:
I changed the order of the two commits. I think it feels better to have them separate since it's two separate fixes. But I agree that with the order changed the WIP-CHANGELOG is more readable